src/App.tsx

```diff
@@ -1,14 +1,87 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { Filter } from './types/EnumFilter';
+import { FiltersType } from './types/Filter';
+import { EyeId } from './types/EyeId';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTOdos] = useState<Todo[]>([]);
```

Comment:
Typographical Error: The state setter function `setFilteredTOdos` is misspelled. It should be `setFilteredTodos` to match the state variable name.